gdal/frmts/adrg/srpdataset.cpp

```diff
@@ -36,8 +36,8 @@
 #include <cstdlib>
 #include <algorithm>
 
-// Uncomment to recognize also .gen files in addition to .img files
-// #define OPEN_GEN
+// Enable option to recognize .gen files in addition to .img files
```

Comment:
I'd prefer if we keep it disabled. This is a bit annoying that the same dataset is recognized by two separate files (thinking of the context of a GUI explorer like QGIS)

Reply:
I don't understand why transmittal header files (.thf) are enabled by default for both formats but .gen files aren't. There shouldn't be a problem as long as the drivers can differentiate between a ADRG and SRP .gen file.

Reply:
My point was that if OPEN_GEN is enabled then when exploring a directory with foo.gen and foo.img from a ASRP/USRP datasets, both files will be recognized, which mean 2 entries in QGIS browser tree for example. Enabling OPEN_GEN should belong to a different change (I don't recall all the rationale for this hesitation between .gen and .img. It has been a long time...)
THF files are a bit different as there might point to several subdatasets.